pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -2810,6 +2810,7 @@ let
       ms-ceintl = callPackage ./language-packs.nix { }; # non-English language packs
 
       ms-dotnettools.csharp = callPackage ./ms-dotnettools.csharp { };
+      ms-dotnettools.csdevkit = callPackage ./ms-dotnettools.csdevkit { };
```

```suggestion
      ms-dotnettools.csdevkit = callPackage ./ms-dotnettools.csdevkit { };
```
Alphabetical order is wrong.

.csh
.csd

d is before h.

---

Done, didn't know this file was in alphabetical order, my bad

---

In attrs and lists default to alphabetical order when inserting itens. There are cases it doesn't follow that, like `meta.broken` (we want broken to be last line to avoid git conflicts when marking packages broken.)